### ISSUE TITLE
[feature request] generate _templates from _templates?

### ISSUE THREAD

**Issue by jrobinson01** [jondot/hygen#152]
I'm building a generator to scaffold a web app. I'd like to be able to place a `_templates` directory inside my `new` directory and have it copied over along with the generated files. Hopefully that makes sense. The goal is to pre-fill a generated project with some useful templates.

**Comment by anithri** [comment]
The same effort that solve #147 solves this one.
Once you can ignore a file or directory, than the shell bits to get them to where you want are pretty straightforward.

**Comment by jrobinson01** [comment]
I'm able to achieve what I want by having templates that generate templates. Here's an example:
```
---
to: packages/<%= h.inflection.dasherize(name) %>/_templates/lit-element/new/lit-element.ejs.t
---
---
to: src/<%%= h.inflection.dasherize(name) %>/<%%= h.inflection.dasherize(name) %>.js
---
import {LitElement, html, css} from 'lit-element';
export default class <%%= h.inflection.classify(name) %> extends LitElement{
  static get styles() {
    return css`

    `;
  }
  static get properties() {

  }
  render() {
    return html`

    `;
  }
};
customElements.define('<%%= h.dasherize(name) %>', <%%= h.inflection.classify(name) %>);

```
It would be great though if there was just a blind 'copy dirs' option but not sure how that would work? Perhaps hygen should ignore (and just copy) anything that doesn't end with the .ejs.t extensions?

**Comment by jondot** [comment]
Closing since should be resolved with #147
